packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,161 @@
+| ------------------------ | -------------------------------------- | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| children                 | `CarouselItem`                         | `undefined`                              | Carousel slides                                                                                                                                                                                              | ✅        |
+| autoPlay                 | `boolean`                              | `false`                                  | If true, the carousel will autoplay                                                                                                                                                                          |          |
+| visibleItems             | `1,2,3,auto`                           | `1`                                      | Total number of carousel items to show at once, if set to 1,2 or 3 all the CarouselItem's width will remain the same but if set to `auto` the carousel items will take up space responsively                 |          |
+| carouselItemWidth        | `ResponsiveProp<width>`                        | `undefined`                              | Sets the width of the carousel items, this can be used with visibleItems: auto to achive automatic bleed                                                                                                     |          |
+| carouselItemAlignment    | `start, end, center`                   | `start`                                  | Sets the align-items CSS property on carousel container which specifies how the carousel items will align if their heights are different                                                                     |          |
+| shouldAddStartEndMargin  | `boolean`                              | `false`                                  | If true, adds extra margin before and after the first/last slides so that they align in center, this prop can be used to achive carousel which is in [GST page](https://razorpay.com/gst-number-search/pan/) |          |
+| showIndicators           | `boolean`                              | `true`                                   | Toggles the visibility of indicators                                                                                                                                                                         |          |
+| showOverlay              | `boolean`                              | `false`                                  | Toggles the visibility of overlay                                                                                                                                                                            |          |
```

Comment:
Should we make this more specific? eg. scroll overlay

Reply:
We might just remove this, we will just have overlayColor, but then we can rename it to `scrollOverlayColor`

Reply:
Removed. and renamed to scrollOverlayColor